frontend/components/shared/Heading/index.jsx

```diff
@@ -50,27 +51,9 @@ export const Heading = ({
     },
     className
   );
-  const headingElement = type => {
-    switch (type) {
-      case HEADING_OPTIONS.TYPE.MAIN_LARGE:
-        return 'h1';
-      case HEADING_OPTIONS.TYPE.MAIN:
-        return 'h1';
-      case HEADING_OPTIONS.TYPE.SECONDARY:
-        return 'h2';
-      case HEADING_OPTIONS.TYPE.SECTION:
-        return 'h3';
-      case HEADING_OPTIONS.TYPE.CARD:
-        return 'h4';
-      case HEADING_OPTIONS.TYPE.CARD_SMALL:
-        return 'h5';
-      default:
-        return 'h1';
-    }
-  };
   return (
     <>
-      {React.createElement(headingElement(type), { className: classes }, [
+      {React.createElement(`${tag ? tag : 'h1'}`, { className: classes }, [
```

Comment:
use anything other than H1 as H1 suppose to be only one per page. I would suggest `p` tag